packages/web3-utils/test/unit/converters.test.ts

```diff
@@ -0,0 +1,210 @@
+import {
+	asciiToHex,
+	bytesToHex,
+	fromWei,
+	hexToAscii,
+	hexToBytes,
+	hexToNumber,
+	hexToNumberString,
+	hexToUtf8,
+	numberToHex,
+	toHex,
+	toNumber,
+	toWei,
+	utf8ToHex,
+} from '../../src/converters';
+import {
+	asciiToHexValidData,
+	bytesToHexInvalidData,
+	bytesToHexValidData,
+	fromWeiInvalidData,
+	fromWeiValidData,
+	hexToAsciiValidData,
+	hexToBytesInvalidData,
+	hexToBytesValidData,
+	hexToNumberInvalidData,
+	hexToNumberValidData,
+	hexToUtf8InvalidData,
+	hexToUtf8ValidData,
+	numberToHexInvalidData,
+	numberToHexValidData,
+	toHexValidData,
+	toWeiInvalidData,
+	utf8ToHexInvalidData,
+	utf8ToHexValidData,
+} from '../fixtures/converters';
+
+describe('converters', () => {
```

Comment:
There aren't tests for

- toDecimal
- fromDecimal
- toUtf8
- fromUtf8
- toAscii
- fromAscii
- hexToString
- stringToHex
- toChecksumAddress

For the above that are simply aliases for other functions, there should still be tests to serve as sanity checks (remember we're maintaining this for the foreseeable future, but there will be developers after us that won't know the assumptions we make today)

Reply:
Sure I will add these tests.